locales/zh_hk.json

```diff
@@ -6,31 +6,31 @@
     "3": "Spark (\u53f2\u5e15)"
   },
   "sizes": {
-    "1": "\u5c0f",
-    "2": "\u77ed",
+    "1": "\u7d30\u5c0f",
+    "2": "\u5c0f",
     "3": "\u6b63\u5e38",
     "4": "\u5927",
     "5": "\u5de8\u5927"
   },
   "types": {
-    "1": "\u6b63\u5e38",
-    "2": "\u6218\u6597",
-    "3": "\u98de",
-    "4": "\u6bd2\u836f",
+    "1": "\u4e00\u822c",
+    "2": "\u683c\u9b25",
+    "3": "\u98db\u884c",
+    "4": "\u6bd2",
     "5": "\u5730\u9762",
-    "6": "\u6447\u6eda",
-    "7": "\u98de\u866b",
-    "8": "\u5e7d\u7075",
-    "9": "\u94a2",
+    "6": "\u5ca9\u77f3",
+    "7": "\u87f2",
+    "8": "\u5e7d\u9748",
+    "9": "\u92fc",
     "10": "\u706b",
     "11": "\u6c34",
     "12": "\u8349",
-    "13": "\u7535",
-    "14": "\u7cbe\u795e",
+    "13": "\u96fb",
+    "14": "\u8d85\u80fd\u529b",
     "15": "\u51b0",
-    "16": "\u9f99",
-    "17": "\u4ed9\u5973",
-    "18": "\u9ed1\u6697"
+    "16": "\u9f8d",
+    "17": "\u60e1",
+    "18": "\u5996\u7cbe"
   },
```

Comment:
Looks like you have a bad rebase, your PR includes commits for other merges.

Reply:
I did nothing more than pull from Dev into new branch and make changes. I didn't rebase anything. Why would pulling latest not include change?

Reply:
When did you pull from Dev to create your branch? Your PR includes 61c81e2 as a commit which was merged into Dev 3 days ago.

Reply:
There was a commit in dev that wasn't in master. I've corrected the issue and rebased the branch since it was a mistake on my part.